src/studio/README.md

```diff
@@ -113,12 +114,18 @@ On MacOS you need two extra steps:
    export ALTINN_KEYS_DIRECTORY=/Users/<yourname>/studio/keys
    ```
 
-Build and run the code.
+Build and prepare for running the application
 
 ```bash
 dotnet build
 yarn run gulp # first time only
```

Comment:
As mentioned in the commend below, this should perhaps be changed to something like

```
yarn run gulp # run this when there are changes in frontend that you want to serve from backend
```

Reply:
@haakemon  I made adjustments. Please have another look

Reply:
Good stuff :)